### PR TITLE
NETOBSERV-868: Update to use cilium auto generated golang structures

### PR DESCRIPTION
**Pull request by msherif1234**
generate golang structure from kernel tc plugin
and update the code to use the new data structures

Also updated cilium pkg to 0.10.0 and updated vendor pkg.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ The Network Observability eBPF Agent allows collecting and aggregating all the i
 egress flows on a Linux host (required a Kernel 4.18+ with eBPF enabled).
 
 * [How to compile](#how-to-compile)
-* [Hot to configure](#how-to-configure)
```
Comment (jotak) [inline README.md:9]: 😂

**Comment by jotak** [comment]
/lgtm
thanks for this PR & the peer review!

**Comment by praveingk** [comment]
Hi @msherif1234 : Thanks for this major cleanup in the types. I tried it out and looks good to me 👍🏽

**Review by praveingk — APPROVED** [review]

**Comment by msherif1234** [comment]
> Hi @msherif1234 : Thanks for this major cleanup in the types. I tried it out and looks good to me 👍🏽

Thanks @praveingk !!

File: README.md (hunk 1 above)
Comment (jpinsonneau) [reply README.md:9]: :fire: :fire: :fire:

**Comment by jotak** [comment]
given the discussion we had today, I've added the  no-qe label

/approve

**Pull request merged**